src/LuaPrinter.ts

```diff
@@ -134,11 +136,15 @@ export class LuaPrinter {
             }
         }
 
-        this.sourceFile = path.basename(sourceFile);
+        this.sourceFile = path.basename(sourceFile, ".ts");
```

Comment:
Won't it fail if there are files with the same name in different directories?
It might be possible to use `debug.getinfo(2).short_src` inside `__TS__SourceMapTraceBack` to make it work without using some fixed path